.github/workflows/scripts/gcp-vm-startup-script.sh

```diff
@@ -11,6 +14,8 @@ sudo cat /etc/ssh/sshd_config
 echo
 echo 'Modifying config:'
 echo 'MaxStartups 500' | sudo tee --append /etc/ssh/sshd_config \
+echo 'TCPKeepAlive no' | sudo tee --append /etc/ssh/sshd_config \
+echo 'ClientAliveCountMax 0' | sudo tee --append /etc/ssh/sshd_config \
```

Comment:
We might want to set this to a high number instead (6 hours), to avoid stuck connections exhausting the connection limit.

Or we can also increase the authenticated connection limit so it's higher than the number of connections we expect from the longest workflow. (About 20.)